README.md

```diff
@@ -321,8 +321,8 @@ have an understanding of the topic yourself, and know what questions to ask.
 
 ### Compensation and equity
 
-🄳 **Compensation** is what you get in exchange for working for a company, including cash
-pay (your salary and bonus, if you get one) and any non-cash pay, including
+🄳 **Compensation** is what employees receive in exchange for working for a company,
```

Comment:
We may need to reference at least parenthetically advisors, contractors, and board members.

Reply:
Agreed that "employees" is too narrow.

Reply:
Addressed.